src/listview.cpp

```diff
@@ -1177,11 +1171,7 @@ QString ListView::refNameAt(const QPoint &pos)
  * Return the device pixel ratio
  */
 qreal ListViewDelegate::dpr(void) const {
-#if QT_VERSION >= QT_VERSION_CHECK(5,6,0)
-    return qApp->devicePixelRatio();
-#else
-    return 1.0;
-#endif
+	return qApp->devicePixelRatio();
```

Comment:
Indentation seems off.

Reply:
not in relation to the context lines, but the sources are inconclusive wrt indentation in several places. I can change it back if you want

Reply:
Oh, wait. In this view Github is showing a tab, when expanding the patch it shows differently giving the impression the indentation was only a couple of spaces. This looks fine giving most of the code is using tabs.